catalog/plugin/catalog-plugin-pdf/src/main/java/ddf/catalog/plugin/pdf/thumbnail/PdfThumbnailPlugin.java

```diff
@@ -0,0 +1,57 @@
+package ddf.catalog.plugin.pdf.thumbnail;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ddf.catalog.data.Metacard;
+import ddf.catalog.transform.pdf.PdfThumbnailGenerator;
+import ddf.content.operation.UpdateResponse;
+import ddf.content.operation.impl.UpdateResponseImpl;
+import ddf.content.plugin.ContentPlugin;
+import ddf.content.plugin.PluginExecutionException;
+import ddf.content.plugin.PostUpdateStoragePlugin;
+
+public class PdfThumbnailPlugin implements PostUpdateStoragePlugin {
```

Comment:
I figured it was not necessary since the metadata and thumbnail are taken care of on ingest. Wouldn't PostCreate be effectively doing it twice?